src/getClientToken.test.ts

```diff
@@ -0,0 +1,33 @@
+import test from "ava";
+import { Response } from "node-fetch";
+import { getClientToken } from "./getClientToken";
+import { DesktopIAPOptions } from "./types";
+
+test("getClientToken", async (t) => {
+  let nFetchCalls = 0;
+
+  const mockFetch = async (): Promise<Response> => {
+    nFetchCalls += 1;
+    return new Response(
+      JSON.stringify({
+        // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
Do we still need this line? I thought we updated eslint rules to disable this error.

Reply:
It's still throwing errors.